package.json

```diff
@@ -32,27 +32,28 @@
     "url": "https://github.com/zendesk/radar_client.git"
   },
   "dependencies": {
-    "engine.io-client": "zendesk/engine.io-client#hhsu/1.4.2-xhr-polling-timeout",
```

Comment:
Can we move away from this patch introduced in https://github.com/zendesk/radar_client/pull/79 ?

Reply:
Actually, the introduced patch was based "loosely" on this [commit](https://github.com/socketio/engine.io-client/commit/3427e744e1c8fcd397dcbb72ea65638ca101d988#diff-04c6e90faac2675aa89e2176d2eec7d8). The commit is merged in the version 1.8.0 of engine.io-client.